test/event_test.py

```diff
@@ -48,7 +48,7 @@
 
     ('UserEvent', pygame.USEREVENT),
     ('UserEvent', pygame.USEREVENT + 1),
-    ('UserEvent', pygame.ACTIVEEVENT - 1),
+    ('UserEvent', pygame.USEREVENT - 1),
```

Comment:
This is probably outside the allowed range? Could be reverted with the change mentioned above.